<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apnSettingsDialog</class>
 <widget class="QDialog" name="apnSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APN configuration</string>
  </property>
  <property name="statusTip">
   <string>Access point name/alias</string>
  </property>
  <property name="whatsThis">
   <string>Use this box to enter the name/alias of access point which the device should connect to</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="9" column="1">
    <widget class="QWidget" name="buttonWrapperWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="useProxyCheckBox">
     <property name="statusTip">
      <string>Enable/disable proxy</string>
     </property>
     <property name="whatsThis">
      <string>Enable/disable using proxy server.</string>
     </property>
     <property name="text">
      <string>Use proxy</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="dns1Label">
     <property name="text">
      <string>DNS 1:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="loginLabel">
     <property name="text">
      <string>Login:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="apnEdit">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="apnLabel">
     <property name="text">
      <string>APN:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="requestPassCheckBox">
     <property name="statusTip">
      <string>Enable/disable password request.</string>
     </property>
     <property name="whatsThis">
      <string>If you check this box, the password will be requested as a condition for successfull network login.</string>
     </property>
     <property name="text">
      <string>Request password</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Address:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Address mode:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="passwordEdit">
     <property name="statusTip">
      <string>Password for network access</string>
     </property>
     <property name="whatsThis">
      <string>This field allows you to save the password for your network login.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="addressModeComboBox">
     <property name="statusTip">
      <string>Address mode of content in &quot;Address&quot; field</string>
     </property>
     <property name="whatsThis">
      <string>This selector allows you to specify the mode of address given in &quot;Address&quot; field. Use IP address mode for &quot;http://XXX.XXX.XXX.XXX/...&quot; format or Text mode for &quot;http://your.url.com/...&quot; format. </string>
     </property>
     <item>
      <property name="text">
       <string>IP address</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text (domain name)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="loginEdit">
     <property name="statusTip">
      <string>Log in name/alias</string>
     </property>
     <property name="whatsThis">
      <string>This field allows you to specify the log in name for getting the access to network</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="dns2Label">
     <property name="text">
      <string>DNS 2:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="dns2Edit">
     <property name="statusTip">
      <string>IP address of DNS2</string>
     </property>
     <property name="whatsThis">
      <string>IP address of secondary DNS server to be used by device.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="dns1Edit">
     <property name="statusTip">
      <string>IP address of DNS1</string>
     </property>
     <property name="whatsThis">
      <string>IP address of primary DNS server to be used by device.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="addressEdit">
     <property name="statusTip">
      <string>Address of target to connect</string>
     </property>
     <property name="whatsThis">
      <string>This field store the target to connect in Text or IP adress mode.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>apnEdit</tabstop>
  <tabstop>loginEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>requestPassCheckBox</tabstop>
  <tabstop>useProxyCheckBox</tabstop>
  <tabstop>addressModeComboBox</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>dns1Edit</tabstop>
  <tabstop>dns2Edit</tabstop>
  <tabstop>acceptButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
